framework/source/class/qx/event/handler/DragDrop.js

```diff
@@ -615,13 +615,16 @@ qx.Class.define("qx.event.handler.DragDrop",
 
       // find current hovered droppable
       var el = e.getTarget();
+      if (this.__startConfig.target === el) {
+        el = document.elementFromPoint(e.getDocumentLeft(), e.getDocumentTop());
+      }
```

Comment:
We already use `elementFromPoint` in the framework here:
https://github.com/qooxdoo/qooxdoo/blob/master/framework/source/class/qx/event/handler/PointerCore.js#L238

The code there uses things like `domEvent.view.document.elementFromPoint` and uses the thouch coordinates of the event to determine the element position.

I don't know why in `qx.event.handler.PointerCore` `domEvent.view.document.elementFromPoint` is used instead of `document.elementFromPoint`, but I think it may be worth investigating. Could it be that in some situation we have `document !== domEvent.view.document`?